src/XIOPAK.CPP

```diff
@@ -1124,7 +1124,18 @@ SEC xfclear(	// Cleans the file by discarting all the data in the file
 	return xf->xflsterr;	/*    bad: report error */
 }
 //=============================================================================
-
+void createPath(			// Joins the directory path with the file name to create its path
```

Comment:
1. Check the name consistency per discussion on other new functions in this file)
2. The description in the comments (and variable names) is more restrictive than the function actual is. This function really just joins two parts of a path. It doesn't have to be a directory and filename.